### DISCUSSION TITLE
Kimai export time sheet for all users

### DISCUSSION THREAD

**Discussion by mxer194** [kimai/kimai#3273] [General]
Hi, I am using the latest version of Kimai Time Tracking for the purpose of tracking my employees' time for various companies. To generate a payroll report, I use time sheets as they allow me to pick a specific time range. Rather than having to export a separate time sheet for each user, I am trying to export times sheets for all users at the same time. 

Older posts suggest that might be possible, but there are no instructions or details to explain how. Can anyone clarify?

**Comment by kevinpapst** [comment]
What about using the reporting view for payroll: /en/reporting/users/month ?

**Reply by kevinpapst** [reply]
See also https://github.com/kevinpapst/kimai2/issues/2559
